examples/discovery-search-app/cypress/integration/collection_filter/spec.ts

```diff
@@ -70,9 +70,9 @@ describe('Collection Filter', () => {
 
         describe('and we click the clear selected collections button', () => {
           beforeEach(() => {
-            cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('postQuery');
+            cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('newPostQuery');
```

Comment:
Naming nitpick: for disco components, we tend to name mocked routes as `{type of request}{request endpoint}{specific name that differentials the request or response}`. So, maybe something like `postQueryClearedSelections` in this case?